UWPInjector/source/main.cpp

```diff
@@ -211,14 +215,69 @@ int main(int argc, char** argv, char** envp)
 
 	std::cout << "Remote Dumper thread found: 0x" << std::hex << IPC::GetTargetThread() << std::endl;
 
+	//get time for filename
+	time_t result = time(NULL);
+	char str[26];
+	ctime_s(str, sizeof str, &result);
+	std::string LogFileName = str;
+	std::replace(LogFileName.begin(), LogFileName.end(), ':', '-');
+	LogFileName.erase(std::remove(LogFileName.begin(), LogFileName.end(), '\n'), LogFileName.end());
+
+	//get proccess name for filename
+	HANDLE hProcess = OpenProcess(PROCESS_ALL_ACCESS | PROCESS_QUERY_INFORMATION |
```

Comment:
```cpp
HANDLE ProcessHandle = OpenProcess(
    ...,
    ...,
    false,
    ...
);
```
would be preferred to keep the code consistent with the rest of the code base. Also `if( blah )` rather than `if (blah)`.